src/test/java/com/google/edith/UserInsightsTest.java

```diff
@@ -238,11 +240,26 @@ public void createJson_withUnkownUser_returnsDefaultJson() {
    * @param quantity quantity 
    * @param date date
    */
-  private void setEntityProperties(Entity entity, String name,
-                                   double price, int quantity, String date) {
-    entity.setProperty("name", name);                                   
+
+  private void setEntityProperties(Entity entity, String name, double price, 
+                                   int quantity, String date) {
+    entity.setProperty("name", name);
     entity.setProperty("price", price);
     entity.setProperty("quantity", quantity);
     entity.setProperty("date", date);
   }
+
+  private boolean compareLists(List<WeekInfo> expected, List<WeekInfo> actual) {
```

Comment:
Would compareOrderedLists be a more appropriate name as it seems if the order is incorrect this method will return false?

Reply:
Yea! I changed it.